packages/tray/test/tray.test.ts

```diff
@@ -100,4 +104,162 @@ describe('Tray', () => {
 
         expect(el.open).to.be.false;
     });
+
+    describe('Dismiss helpers', () => {
+        it('renders visually-hidden, keyboard-accessible dismiss helpers when no buttons detected', async () => {
+            const el = await fixture<Tray>(html`
+                <sp-tray open>
+                    <sp-menu style="width: 100%">
+                        <sp-menu-item>Item 1</sp-menu-item>
+                        <sp-menu-item>Item 2</sp-menu-item>
+                    </sp-menu>
+                </sp-tray>
+            `);
+            await elementUpdated(el);
+
+            const helpers = el.shadowRoot.querySelectorAll('.visually-hidden');
+            expect(helpers.length).to.equal(2);
+
+            const buttons = el.shadowRoot.querySelectorAll(
+                '.visually-hidden button'
+            );
+            expect(buttons.length).to.equal(2);
+            expect(buttons[0].getAttribute('aria-label')).to.equal('Dismiss');
+            expect(buttons[0].getAttribute('tabindex')).to.be.null;
```

Comment:
Let's test this by tabbing to the button, pressing `Enter` and checking if the tray closes. It is a more accurate way to test if this works.

In rthe PR description, we should also ask reviews to test with a keyboard as well.